sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -1186,3 +1218,122 @@ def _check_consistency(self, y_test, y_pred):
 
         # check consistency of forecast hierarchy with training data
         assert set(y_test.index.names).issubset(y_pred.index.names)
+
+
+def _get_expected_columns(y_test):
+    if isinstance(y_test, pd.Series):
+        if hasattr(y_test, "name") and y_test.name is not None:
+            return [y_test.name]
+        else:
+            return [0]
+    else:
+        return y_test.columns
+
+
+def _check_predict_intervals(pred_ints, y_test, fh, coverage):
+    """Check expected interval prediction output."""
+    # check expected type
+    valid, msg, _ = check_is_mtype(
+        pred_ints,
+        mtype="pred_interval",
+        scitype="Proba",
+        return_metadata=True,
+        msg_return_dict="list",
+    )  # type: ignore
+    assert valid, msg
+
+    # check index (also checks forecasting horizon is more than one element)
+    cutoff = get_cutoff(y_test, return_index=True)
+    index_pred = pred_ints.iloc[
```

Comment:
I see now where the change is.

Then, could you kindly use `ForecastingHorizon.get_expected_pred_idx` instead, which is the single interface point for getting the expected index to be returned by predict-like methods?

Reply:
updated `_assert_correct_pred_time_index` to use `ForecastingHorizon.get_expected_pred_idx`.